website/docs/workloads/mlperf/mlperf.md

```diff
@@ -1,40 +1,67 @@
-* [MLPerf Training Documentation](https://github.com/mlcommons/training_results_v2.0/blob/main/MLPerf%E2%84%A2%20Training%20v2.0%20Results%20Discussion.pdf)  
+# MLPerf
+
+MLPerf is a consortium of AI leaders from academia, research labs, and industry whose mission is to “build fair and useful benchmarks” that provide unbiased evaluations of training and inference performance for hardware, software, and services—all conducted under prescribed conditions. To stay on the cutting edge of industry trends, MLPerf continues to evolve, holding new tests at regular intervals and adding new workloads that represent the state of the art in AI.
+
+* [MLPerf Training Documentation](https://github.com/mlcommons/training_results_v2.1/blob/main/MLPerf%E2%84%A2%20Training%20v2.0%20Results%20Discussion.pdf)  
 * [MLPerf Inference Documentation](https://github.com/mlcommons/inference_results_v2.0)  
-* [MLPerf Training Benchmarks](https://github.com/mlcommons/training_results_v2.0/tree/main/NVIDIA/benchmarks)
+* [MLPerf Training Benchmarks](https://github.com/mlcommons/training_results_v2.1/tree/main/NVIDIA/benchmarks)
 * [MLPerf Inference Benchmarks](https://github.com/mlcommons/inference_results_v2.0/tree/master/closed/NVIDIA)
+* [MLPerf Training Bert Preprocessing Data](./mlperf-trainining-bert-preprocessing-data.md)
 
 ## System Requirements
+
 This is a GPU-specific workload and requires high-performance graphic cards to run. It is recommended that the system-under-test have a high-performing Nvidia (e.g. M60 or higher) or AMD (e.g. MI25 or higher)
 graphics card.
 
 ## Supported Hardware Systems
-The following section defines the hardware systems/SKUs on which the MLPerf 2.0 workload will run effectively in cloud environments. These hardware systems contain
+
+The following section defines the hardware systems/SKUs on which the MLPerf workload will run effectively in cloud environments. These hardware systems contain
 GPU components for which the MLPerf workload is designed to test.
 
-* **Datacenter systems**  
+* **Datacenter systems MLPerf Inference**  
   * A100-SXM-80GBx8 (NVIDIA DGX A100, 80GB variant)
-  * A100-SXM-80GBx4 (NVIDIA DGX Station A100, "Red October", 80GB variant)
+  * A100-SXM-80GBx4 (NVIDIA DGX Station A100, "Red SEPTober", 80GB variant)
   * A100-PCIex8 (80GB variant)
   * A2x2
   * A30x8
 
-* **Edge Systems**
+* **Edge Systems MLPerf Inference**
   * A100-SXM-80GBx1
   * A100-PCIex1 (80 GB variant)
   * A30x1
   * A2x1
   * Orin
   * Xavier NX
 
-Additional details on whether a system is supported or not can be found in the documetation here: 
+* **Supported Config Files for MlPerf Bert Training (config_{nodes}x{gpus per node}x{local batch size}x{gradien accumulation}.sh)**
+  * config_A30_1x2x224x14.sh
```

Comment:
These shell scripts, are they direct port from nvidia? or you modified it and only leave the bert section?

Reply:
These are created by Nvidia
& these are for Bert only

Reply:
Added the source link below the list.
In order to know the list of Configuration available and their source